iterative/utils/logger.go

```diff
@@ -106,7 +106,8 @@ func TpiLogger(d *schema.ResourceData) *logrus.Entry {
 }
 
 func hideUnwantedPrefix(levelText, newPrefix, message string) string {
-	unwantedPrefixLength := len(fmt.Sprintf("yyyy-mm-ddThh:mm:ss.mmmZ [%s] provider.terraform-provider-iterative: [%[1]s]", levelText))
+	timeString := time.Now().Format("2006-01-02T15:04:05.000Z0700")
```

Comment:
Special day?

Reply:
Yes, see [the documentation](https://pkg.go.dev/time#pkg-constants)

> _The reference time used in these layouts is the specific time stamp:_
> 
>     01/02 03:04:05PM '06 -0700
>
> _(January 2, 15:04:05, 2006, in time zone seven hours west of GMT). That value is recorded as the constant named Layout, listed below. As a Unix time, this is 1136239445. Since MST is GMT-0700, the reference would be printed by the Unix date command as:_
> 
>     Mon Jan 2 15:04:05 MST 2006
>
> _It is a regrettable historic error that the date uses the American convention of putting the numerical month before the day._